ansible/projects/ovn-kubernetes/deploy-project.yaml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Deploy ovn-kubernetes project
+  hosts: kind_nodes
+  become: true
+  remote_user: root
+  vars:
+    ovn_image:  ovn-kube-f
+    ovn_k8s_deployments: [ovn-setup, ovs-node, ovnkube-db, ovnkube-master, ovnkube-node]
+  tasks:
+    - name: Create temp directory for ovn-kuberentes
+      tempfile:
+        state: directory
+        suffix: _ovn-kubernetes
+      register: ovn_kubernetes_dir
+    - name: Clone ovn-kubernetes
+      git:
+        repo: https://github.com/ovn-org/ovn-kubernetes
+        dest: "{{ ovn_kubernetes_dir.path }}"
+      when: 'pull_request == ""'
+    - name: Clone ovn-kubernetes with pull request {{ pull_request }}
+      git:
+        repo: https://github.com/ovn-org/ovn-kubernetes
+        dest: "{{ ovn_kubernetes_dir.path }}"
+        refspec: "+refs/pull/{{ pull_request }}/head:{{ pull_request }}"
+        version: "{{ pull_request }}"
+      when: 'pull_request != ""'
+    - name: Build ovn-kubernetes image
+      make:
+        chdir: "{{ ovn_kubernetes_dir.path }}/dist/images"
+        target: fedora
+    - name: Load image into cluster
```

Comment:
I think it would be better to move this to somewhere global where all projects will be able to use, see reusable tasks in https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse.html

Reply:
Updated

Reply:
@Mmduh-483  Thanks for the update, one last comment regarding this, i think it would be good if the tasks yaml files are separated from the main playbook, basically i think it is would great if the "global tasks" are put inside a standalone folder. This because these can populate very quickly and directory would become a miss